hmda/Jenkinsfile

```diff
@@ -38,13 +38,15 @@ volumes: [
         withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'dockerhub',
                   usernameVariable: 'DOCKER_HUB_USER', passwordVariable: 'DOCKER_HUB_PASSWORD']]) {
             if (gitTagged || gitBranch == "master") {
```

Comment:
Let's include `isDelpoyPr` in this.

Reply:
This block doesn't deploy so it would be redundant. We never deploy from docker hub block anymore.

Reply:
Got you, looks good then.